### PR TITLE
systemwide section added: seo, smo & gdpr

### PR DESCRIPTION
**Pull request by MonikaMrozek**

### CODE REVIEW

File: docs/dashboard/customers.md [hunk 1]
```diff
@@ -67,22 +67,15 @@ Users are also able to request their account to be deleted, from the storefront
...
-> 
-> ##### Cookies
->  
-> All cookies used by Saleor are strictly necessary to move around the website and use its features, therefore there is no need to notify the users about them.
-> 
-> ##### Privacy Policy and Terms of Service statements
 >
-> Make sure your _Terms of Service_ and _Privacy Policy_ statements properly communicate to your users who you are and how you are using their data. 
-We recommend that you ensure your policies are kept up to date and are clear to your readers.
+> For more details on data protection implementation in Saleor, see our [GDPR](dasboard/systemwide/gdpr.md) topic.
```
Comment (maarcingebala) [inline docs/dashboard/customers.md]: Typo in link: dasboard

File: docs/dashboard/customers.md [hunk 2]
```diff
@@ -67,22 +67,15 @@ Users are also able to request their account to be deleted, from the storefront
...
-> 
-> ##### Cookies
->  
-> All cookies used by Saleor are strictly necessary to move around the website and use its features, therefore there is no need to notify the users about them.
-> 
-> ##### Privacy Policy and Terms of Service statements
 >
-> Make sure your _Terms of Service_ and _Privacy Policy_ statements properly communicate to your users who you are and how you are using their data. 
-We recommend that you ensure your policies are kept up to date and are clear to your readers.
+> For more details on data protection implementation in Saleor, see our [GDPR](dashboard/systemwide/gdpr.md) topic.
```
Comment (NyanKiyoshi) [inline docs/dashboard/customers.md]:
```suggestion
> For more details on data protection implementation in Saleor, see our [GDPR](dashboard/systemwide/gdpr) topic.
```

File: docs/dashboard/systemwide/introduction.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+id: introduction
+title: Systemwide Features
+sidebar_label: Introduction
+---
+This section describes features common to many or all sections of the dashboard.
+
+[card-grid]
+[**GDPR** How GDPR regulations are handled in Saleor.](dashboard/systemwide/gdpr.md)
```
Comment (NyanKiyoshi) [inline docs/dashboard/systemwide/introduction.md]:
```suggestion
[**GDPR** How GDPR regulations are handled in Saleor.](dashboard/systemwide/gdpr)
```

File: docs/dashboard/systemwide/introduction.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+[**SEO and SMO** A short guide to SEO and SMO with quick tips to increase traffic to your site.](dashboard/systemwide/seo-smo.md)
```
Comment (NyanKiyoshi) [inline docs/dashboard/systemwide/introduction.md]:
```suggestion
[**SEO and SMO** A short guide to SEO and SMO with quick tips to increase traffic to your site.](dashboard/systemwide/seo-smo)
```

File: docs/dashboard/catalog/products.md [hunk 5]
```diff
@@ -180,7 +134,7 @@ Add the name of your product and a description. Use text formatting to make your
 
 Type the price of your product manually or using the stepper by the currency indicator. If your product is tax-exempt, leave the [[Charge&nbsp;taxes&nbsp;for&nbsp;this&nbsp;item]] checkbox empty. If you need to charge taxes, click on the box and a green check mark will appear.
 
-For more information on taxes, see the [Taxes](configuration/taxes) topic.
+For more information on taxes, see the [Taxes](configuration/taxes.md) topic.
```
Comment (NyanKiyoshi) [inline docs/dashboard/catalog/products.md]:
```suggestion
For more information on taxes, see the [Taxes](configuration/taxes) topic.
```

**Comment by maarcingebala** [comment]
> Looks good to me. But the links are supposed to be without `.md` I believe?

@NyanKiyoshi I've already merged it but it seems that we do use `.md`. We will check tomorrow and fix the links if they got broken.

**Comment by MonikaMrozek** [comment]
Checked it and indeed we do need the .md. I've reinstated the extension.

**Pull request merged**